packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -265,6 +265,12 @@ class _TooltipOverlay extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    ThemeData theme = Theme.of(context);
+    ThemeData darkTheme = new ThemeData(
+      brightness: Brightness.dark,
+      accentColor: theme.accentColor,
```

Comment:
Missed this. Sent a #6595 to kill the accent colour & accent colour brightness now that I'm no longer applying a Theme. Serves me right for leaving this 4 days before re-picking it up :(